airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_incremental.py

```diff
@@ -337,7 +337,7 @@ async def test_state_with_abnormally_large_values(
                 else:
                     assert self._check_cursor_by_char_types(
                         actual_cursor_value, future_state_cursor_value
-                    ), "Actual and future state formats do not match. Actual cursor value: {actual_cursor_value}, future cursor value: {future_state_cursor_value}"
+                    ), f"Actual and future state formats do not match. Actual cursor value: {actual_cursor_value}, future cursor value: {future_state_cursor_value}"
```

Comment:
Nice catch! That being said though, this change will need a release of the CATs framework. I don't know if we can do both in the same PR. I think you can because I remember @alafanechere mentioning that this CATs image is built before the each tests and hence the version number is more for documentation purposes. Can we:
* Get a confirmation from @alafanechere ?
* Still up the version for documentation purposes?

Reply:
@alafanechere There is more to add to CAT, after the recent changes, see this comment as well, please:
https://github.com/airbytehq/airbyte/pull/39136#issuecomment-2158006606